src/main/java/io/openliberty/tools/intellij/LibertyExplorer.java

```diff
@@ -60,6 +69,8 @@ public static Tree buildTree(Project project, Color backgroundColor) {
         String projectName = null;
```

Comment:
Since `projectName` is used inside each of the `for (PsiFile file : mavenBuildFiles) {` and `for (PsiFile file : gradleBuildFiles) {` loops and changes between iterations, it's better to declare the variable within the scope of each of those loops instead of here.  Otherwise it gives the impression that this variable is needed outside of the loops.